lib/schema/union.js

```diff
@@ -14,12 +14,12 @@ const firstValueSymbol = Symbol('firstValue');
  */
 
 class Union extends SchemaType {
-  constructor(key, options, schemaOptions = {}) {
+  constructor(key, options, schemaOptions, parentSchema) {
```

Comment:
This change should likely be documented somehwere in the code, though i dont know if this already exists somehwere (and `SchemaType` itself seems to have different parameters, so likely not there).
Also the "Custom Schema Types" guide seems to be somewhat outdated, and this change should likely be documented there as a available parameter (`/docs/customschematypes.md`).

Reply:
Fixed, although I did a lot of refactor to make this consistent